study-builder/studies/atcp/medical-history.conf

```diff
@@ -0,0 +1,6111 @@
+
+          ],
+          "blockType": "GROUP",
+          "shownExpr": null
+        }
+      ]
+    },
+    {
+      "nameTemplate": null,
+      "icons": [],
+      "blocks": [
+        {
+          "listStyleHint": "BULLET",
+          "title": null,
+          "nested": [
+            {
+              "question": {
+                include required("../snippets/bool-question-yes-no-required.conf"),
+                "hideNumber": true,
+                "promptTemplate": {
+                  "templateType": "HTML",
+                  "templateText": """
+                    <p>$diagnosed_p1 $ddp.participantFirstName() $diagnosed_p2</p>
+                    <em class="secondary-text bold">$diagnosed_note</em>
+                  """,
+                  "variables": [
+                    {
+                      "name": "diagnosed_p1",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.medical_history_diagnosed_p1}
+                        }
+                      ]
+                    },
+                    {
+                      "name": "diagnosed_p2",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.medical_history_diagnosed_p2}
+                        }
+                      ]
+                    },
+                    {
+                      "name": "diagnosed_note",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.medical_history_diagnosed_note}
+                        }
+                      ]
+                    }
+                  ]
+                }
+                "stableId": "DIAGNOSED"
+              },
+              "blockType": "QUESTION",
+              "shownExpr": null
+            }
+          ],
+          "blockType": "GROUP",
+          "shownExpr": null
+        }
+      ]
+    },
+    {
+      "nameTemplate": null,
+      "icons": [],
+      "blocks": [
+        {
+          "listStyleHint": "BULLET",
+          "title": null,
+          "nested": [
+            {
+              "question": {
+                include required("../snippets/numeric-question.conf"),
+                "hideNumber": true,
+                "promptTemplate": {
+                  "templateType": "HTML",
+                  "templateCode": null,
+                  "templateText": """
+                      $years_p1 $ddp.participantFirstName() $years_p2
+                  """,
+                  "variables": [
+                    {
+                      "name": "years_p1",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.medical_history_age_years_p1}
+                        }
+                      ]
```

Comment:
Might or might not make sense, but we have composite questions that let you combine as one question multiple simpler questions. You have the option of just having a single composite answer (which is what you would do here if you chose to use it)  or allowing the user to add as many responses as user wants. This might be a useful example:
https://github.com/broadinstitute/ddp-study-server/blob/12ce673b3d6a4eafaa312af3d84e3004aa87517a/study-builder/studies/brain/about-you.conf#L176